rpc/common/service.proto

```diff
@@ -212,15 +212,37 @@ message DetectedLicense {
   string          name       = 5;
   float           confidence = 6;
   string          link       = 7;
+}
+
+enum LicenseCategory {
+  LICENSE_CATEGORY_UNSPECIFIED  = 0;
```

Comment:
The generated constants are now like `LicenseCategory_LICENSE_CATEGORY_UNSPECIFIED`. Do we need `LICENSE_CATEGORY_` prefix for these enums?

Reply:
@knqyf263 A conflict can happen within a single proto file without using the enum name prefix. https://github.com/protocolbuffers/protobuf/issues/5425

Also protobuf [recommends](https://protobuf.dev/programming-guides/style/#enums) using prefix for enum names.
There is an open [issue](https://github.com/golang/protobuf/issues/513) to remove the prefix in the enum name.

Reply:
As people are complaining in the issue, it looks really weird.

>I found the following https://github.com/golang/protobuf/issues/513#issuecomment-1672772801 to improve the readability of enumerations:

This is not ideal, but we can accept it. @nikpivkin Can you please rewrite the enum in the way and write a comment on why we need this workaround?

Reply:
Done https://github.com/aquasecurity/trivy/pull/5382/commits/2096c91d71aa4349c38593e6d7a7a109d71057d3

Reply:
Thanks!